Author: Andreas Tille <tille@debian.org>
Last-Update: 2020-07-14
Bug-Debian: https://bugs.debian.org/965032
Description: Make sure Python3 is used
Forwarded: not-needed

Index: bcftools/misc/guess-ploidy.py
===================================================================
--- bcftools.orig/misc/guess-ploidy.py
+++ bcftools/misc/guess-ploidy.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Plot the output of "bcftools +guess-ploidy -v"
 #
@@ -24,7 +24,7 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 
-from __future__ import print_function
+
 
 import matplotlib as mpl
 mpl.use('Agg')
