Author: Andreas Tille <tille@debian.org>
Last-Update: Tue, 14 Jul 2020 22:21:56 +0200
Bug-Debian: https://bugs.debian.org/965032
Description: Make sure Python3 is used

--- a/misc/guess-ploidy.py
+++ b/misc/guess-ploidy.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Plot the output of "bcftools +guess-ploidy -v"
 #
@@ -24,7 +24,7 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 
-from __future__ import print_function
+
 
 import matplotlib as mpl
 mpl.use('Agg')
