Description: Set installation prefix
Author: Afif Elghraoui <afif@ghraoui.name>
Forwarded: not-needed
Last-Update: 2015-11-03
--- bcftools.orig/Makefile
+++ bcftools/Makefile
@@ -57,7 +57,7 @@
     GSL_LIBS = -lgsl -lgslcblas
 endif
 
-prefix      = /usr/local
+prefix      = /usr
 exec_prefix = $(prefix)
 bindir      = $(exec_prefix)/bin
 libdir      = $(exec_prefix)/lib
@@ -65,7 +65,7 @@
 mandir      = $(prefix)/share/man
 man1dir     = $(mandir)/man1
 
-plugindir   = $(libexecdir)/bcftools
+plugindir   ?= $(libexecdir)/bcftools
 pluginpath  = $(plugindir)
 
 MKDIR_P = mkdir -p
