Description: Set installation prefix
Author: Afif Elghraoui <afif@ghraoui.name>
Forwarded: not-needed
Last-Update: 2015-11-03
--- a/Makefile
+++ b/Makefile
@@ -41,7 +41,7 @@ OBJS     = main.o vcfindex.o tabix.o \
            mpileup.o bam2bcf.o bam2bcf_indel.o bam_sample.o \
            ccall.o em.o prob1.o kmin.o # the original samtools calling
 
-prefix      = /usr/local
+prefix      = /usr
 exec_prefix = $(prefix)
 bindir      = $(exec_prefix)/bin
 libdir      = $(exec_prefix)/lib
--- a/config.mk.in
+++ b/config.mk.in
@@ -47,7 +47,7 @@ GSL_LIBS = -lgsl -lgslcblas
 
 PLATFORM   = @PLATFORM@
 PLUGINS_ENABLED = @enable_bcftools_plugins@
-plugindir = @bcf_plugindir@
+plugindir ?= @bcf_plugindir@
 PLUGIN_EXT = @PLUGIN_EXT@
 
 @Hsource@BGZIP = /usr/bin/bgzip
