Description: Use packaged libraries
Author: Afif Elghraoui <afif@ghraoui.name>
Forwarded: not-needed
Last-Update: 2015-11-03
--- bcftools.orig/Makefile
+++ bcftools/Makefile
@@ -29,17 +29,14 @@
 all: $(PROG) $(TEST_PROG)
 
 # Adjust $(HTSDIR) to point to your top-level htslib directory
-HTSDIR = ../htslib
-include $(HTSDIR)/htslib.mk
-HTSLIB = $(HTSDIR)/libhts.a
-BGZIP  = $(HTSDIR)/bgzip
-TABIX  = $(HTSDIR)/tabix
+BGZIP  = /usr/bin/bgzip
+TABIX  = /usr/bin/tabix
 
 CC       = gcc
 CPPFLAGS =
 CFLAGS   = -g -Wall -Wc++-compat -O2
-LDFLAGS  =
-LIBS     =
+LDFLAGS  ?=
+LIBS     += $(LDFLAGS) -lhts
 
 OBJS     = main.o vcfindex.o tabix.o \
            vcfstats.o vcfisec.o vcfmerge.o vcfquery.o vcffilter.o filter.o vcfsom.o \
@@ -57,7 +54,7 @@
 ifdef USE_GPL
     EXTRA_CPPFLAGS += -DUSE_GPL
     OBJS += polysomy.o peakfit.o
-    GSL_LIBS = -lgsl -lcblas
+    GSL_LIBS = -lgsl -lgslcblas
 endif
 
 prefix      = /usr/local
@@ -101,10 +98,10 @@
 .c.o:
 	$(CC) $(CFLAGS) $(EXTRA_CPPFLAGS) $(CPPFLAGS) -c -o $@ $<
 
-test: $(PROG) plugins test/test-rbuf $(BGZIP) $(TABIX)
+test: $(PROG) plugins test/test-rbuf
 	./test/test.pl --exec bgzip=$(BGZIP) --exec tabix=$(TABIX)
 
-test-plugins: $(PROG) plugins test/test-rbuf $(BGZIP) $(TABIX)
+test-plugins: $(PROG) plugins test/test-rbuf
 	./test/test.pl --plugins --exec bgzip=$(BGZIP) --exec tabix=$(TABIX)
 
 
@@ -180,8 +177,8 @@
 test/test-rbuf: test/test-rbuf.o
 	$(CC) $(LDFLAGS) -o $@ $^ -lm $(LIBS)
 
-bcftools: $(HTSLIB) $(OBJS)
-	$(CC) -rdynamic $(LDFLAGS) -o $@ $(OBJS) $(HTSLIB) -lpthread -lz -lm -ldl $(GSL_LIBS) $(LIBS)
+bcftools: $(OBJS)
+	$(CC) -rdynamic $(LDFLAGS) -o $@ $(OBJS) -lpthread -lz -lm -ldl $(GSL_LIBS) $(LIBS)
 
 doc/bcftools.1: doc/bcftools.txt
 	cd doc && a2x -adate="$(DOC_DATE)" -aversion=$(DOC_VERSION) --doctype manpage --format manpage bcftools.txt
